### DISCUSSION TITLE
Monthly Meeting: 2. November 2021

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#1519] [Show and tell]
# Protocol

- `opacity` in `BaseSymbolizer`
  - clarify meaning in UI and parsers (parser feedback)
  - SLD-Parser ignore or multiply global opacity with Fill- Stroke-Opacity?
  - continue discussion here -> https://github.com/geostyler/geostyler/discussions/1510

- Layout for usage in an APP
  - Currently draft. Work continues on friday. #nextplayday

- Talks and Videos will be collected on the geostyler.org page ("In the media" ?)

- Usage of geostyler-style `WriteResult` and `ReadResult` should be extended in all parsers and the geostyler ui.
  - Show infos about unsupportedProperties, …

- Schedule the codesprint

**Comment by chrismayer** [comment]
PR https://github.com/geostyler/geostyler/pull/1505/ is one major part of the above mentioned draft for a new `"Layout for usage in an APP"`, right? Are there more PRs / branches, which refer to this? Are the shown mockups in the meeting accessible somewhere?

**Reply by jansule** [reply]
All work is currently going into branch [geostyler/style-cards](https://github.com/geostyler/geostyler/tree/style-cards). There you can checkout the new components, until the feature is complete. For every feature, there is at least one example in the docs already.

The related PRs can be found [here](https://github.com/geostyler/geostyler/pulls?q=is%3Apr+base%3Astyle-cards+).

Today I am also going to continue working on this.

> Are the shown mockups in the meeting accessible somewhere?

Currenlty not, but we can try to make it accessible (at least via screenshots)

**Comment by chrismayer** [comment]
Thanks for the reply and the infos @jansule :+1: 

>> Are the shown mockups in the meeting accessible somewhere?

> Currenlty not, but we can try to make it accessible (at least via screenshots)

Would be very cool. Thanks in advance

**Comment by jansule** [comment]
Please note that the surrounding container (the one with the `X` in the upper right corner) is not part of GeoStyler, but the component where GeoStyler can be embedded in. So this can be a side panel, bottom panel, or anything else.

Style Default Overview

![Style_Default_Overview](https://user-images.githubusercontent.com/12186477/140479674-5cfd9171-7d1a-4c5c-bbde-3124c5fd876a.png)

Style Select

![Style_Select](https://user-images.githubusercontent.com/12186477/140479676-02e263a8-608b-404b-9eb5-b2b73d73e77d.png)

Multi Edit View

![Multi_Edit_View](https://user-images.githubusercontent.com/12186477/140479672-3d0d29d1-6e71-4538-bc62-2133f6cdb804.png)

Rule Default Overview

![Style_Rule_Default_Overview](https://user-images.githubusercontent.com/12186477/140479675-20ba6441-c19a-4124-bc7b-64dbd64d876a.png)

Filter View
![Filter_View](https://user-images.githubusercontent.com/12186477/140479668-9975737d-8ddd-49b0-bc2a-c5215a727612.png)
